lib/workers/repository/init/inherited.ts

```diff
@@ -99,5 +100,43 @@ export async function mergeInheritedConfig(
       'Removed global config from inherited config.',
     );
   }
-  return mergeChildConfig(config, filteredConfig);
+
+  if (is.nullOrUndefined(filteredConfig.extends)) {
+    return mergeChildConfig(config, filteredConfig);
+  }
+
+  let returnConfig = filteredConfig;
```

Comment:
do we really need another variable? can't we rename and reuse `filteredConfig`?

Reply:
I think using `returnConfig` is more readable and cleaner. Reusing same var might get bit confusing to track.

Reply:
On second thought, even having too many vars can be confusing.